src/main/java/carracinggame/domain/Car.java

```diff
@@ -0,0 +1,50 @@
+package carracinggame.domain;
+
+public class Car {
+
+    public static final int INITIAL_POSITION = 0;
+    public static final int REFERENCE_NUMBER = 4;
+
+    private String name;
+    private int position = INITIAL_POSITION;
+
+    Car(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public int getPosition() {
+        return this.position;
+    }
+
+    public void move(int number) {
+        if (isOverProbability(number)) {
+            position++;
+        }
+    }
+
+    private boolean isOverProbability(int number) {
+        if (number >= REFERENCE_NUMBER) {
+            return true;
+        }
+        return false;
+    }
+
+    public boolean isWinner(Car comparisonCar) {
```

Comment:
```suggestion
    public boolean isWinner(Car car) {
```

굳이 인자의 이름이 비교 자동차일 필요는 없다고 생각해요. 그 이유는 이미 내부에서 자동차와 비교한다는 사실을 알고 있으니까요! 불필요한 정보를 제공한다고 생각합니다 😀

Reply:
그러네요 .ㅎㅎ 불필요한 정보이군요.

Reply:
네, 메서드 내에서 사용되는 변수 하나하나의 이름에도 의미가 있어요. 이를 유의하면 좋을 것 같아요 !